lib/iris/fileformats/ugrid_cf_reader.py

```diff
@@ -220,15 +220,19 @@ def complete_ugrid_cube(self, cube):
                 " "
             ):
                 node_var = self.dataset.variables[node_var_name]
-                node_coordinates.append(node_var.getncattr("standard_name"))
-            node_coordinates = set(node_coordinates)
+                name = (
+                    getattr(node_var, "standard_name", None)
+                    or getattr(node_var, "long_name", None)
+                    or node_var_name
+                )
+                node_coordinates.append(name)
```

Comment:
`node_coordinates` is no longer a set. Should #L79-80 therefore be changed also?
https://github.com/SciTools/iris/blob/d8274aeb4621327366faae81f48baeda50b82509/lib/iris/fileformats/ugrid_cf_reader.py#L79

Reply:
Forgot that! thank you!